app/views/topics/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h2>Topics</h2>
+<p>
+  This is where the topics go...
+
+  <% @topics.each do |topic| %>
+    <tr>
+      <td><%= topic.body %></td>
+    </tr>
+  <% end %>
+</p>
```

Comment:
Files should always end with a line ending, which is why github is complaining here.
(You can configure sublime to add this for you automatically)